SmartSyncExplorerSwift/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "SmartSyncExplorerSwift",
+    "version": "0.0.1",
+    "private": true,
+    "sdkDependencies": {
+        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#v6.1.0"
```

Comment:
Point to dev. On release day, we will point the master branch version to the right tag.

Reply:
Change this to `dev`.